dependencies/dependencies.ini

```diff
@@ -1,6 +1,9 @@
 [runtime]
 # 'python' key expects a value in the Python requirements file format
 #  https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format 
-python=numpy==1.17.5
-       cython
+python=
 
+[compilation]
+python=numpy
+       cython
+       torch==1.7.1
```

Comment:
I introduced the `compilation` section for python because some pip installations might include compiling source code, e.g., [inplace_abn](https://github.com/mapillary/inplace_abn): "_Note that some parts of InPlace-ABN have native C++/CUDA implementations, meaning that the command above will need to compile them._"

However, you are right that this is merely done to deal with dependencies that have to be installed one after another and only applies when actually running the toolkit. Nonetheless, I do not think it is good to hide such dependencies directly in the source code of _install.sh_.

I checked the other dependencies and `inplace_abn` is not the only package with those requirements.
- `cocoapi` requires Cython to be installed in a previous step
- `DCNv2` also requires Python to be installed in a previous step

Reply:
Ok. But still the `compilation` section is not the correct section where to put them. 

We have different options (that don't exclude each other):
1) to first install the global dependencies specified in `dependencies/dependencies.ini` and in a separate instruction install the rest: only generic dependencies like `torch` should be listed here but not specific ones used only by 1/2 tools
2) add a new label in the `dependency.ini` (https://github.com/tasostefas/opendr_internal/wiki/Adding-New-Tool#dependencies) for example `python-dependencies` where for each module we can specify some packages that needs to be installed before others.

Reply:
Those are good ideas. Personally, I would combine both of the suggestions and also add checks preventing the work packages from overwriting the global dependencies.

I should be able to integrate those changes this afternoon.

Reply:
I integrated your proposed changes. Unfortunately, I couldn't think of an easy (yet proper) way to take care of the version parsing of python packages and prevent globally specified versions from being overwritten. Since it's not very urgent right now, I would postpone the idea to another time.